images.CI/macos/select-datastore.ps1

```diff
@@ -36,7 +36,9 @@ param(
     [ValidateNotNullOrEmpty()]
     [string]$VIPassword,
 
-    [string]$TagCategory = "Busy"
+    [string]$TagCategory = "Busy",
+
+    [string]$Cluster
```

Comment:
Add and pass $cluster parameter to Select-DataStore function

Reply:
Updated